docker/Dockerfile.alpine

```diff
@@ -1,5 +1,5 @@
 FROM node:alpine
-MAINTAINER Coderaiser
+LABEL maintainer="Coderaiser"
```

Comment:
What difference between `MAINTEINER` and `LABEL maintainer` :)?

Reply:
it is now [deprecated](https://docs.docker.com/engine/reference/builder/#maintainer-deprecated) to use the `MAINTAINER` entry. You should use `LABEL` and reflect that information there.